src/main/java/frc/robot/subsystems/AlgaeEndEffector.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import com.revrobotics.spark.SparkLimitSwitch;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class AlgaeEndEffector extends SubsystemBase {
+    static final int MOTOR_CURRENT_LIMIT = 30;
+    static final double MOTOR_VOLTAGE_COMP = 10; //This sets a limit for voltage to 10 so it is repeatable untill the battery dips below 10 volts
+   
+    // TODO: Set these to real speeds
+    static final double INTAKE_SPEED = -1;
+    static final double PROCESSOR_SPEED = 1;
+    static final double BARGE_SPEED = 3;
```

Comment:
Speeds are -1 -> 1 so "3" is not possible.  I would set these to +/- 0.5 for now and we will adjust with testing.